packages/server/core/src/base/utils/serverConfig.ts

```diff
@@ -1,12 +1,14 @@
-import * as path from 'path';
 import { compatRequire, fs, DEFAULT_SERVER_CONFIG } from '@modern-js/utils';
-import type { ServerConfig, ServerOptions } from '@modern-js/server-core';
 import mergeDeep from 'merge-deep';
+import { ServerConfig } from '../../core/plugin';
+import { ServerOptions } from '../../types/config';
 
-export const getServerConfigPath = (
+export const getServerConfigPath = async (
   distDirectory: string,
   serverConfigFile: string = DEFAULT_SERVER_CONFIG,
 ) => {
+  // Only support node.js
+  const path = await import('path');
```

Comment:
It's possible, but we try to avoid that, and ideally we only have base

Reply:
Yes, this is our expectation. But now serverConfig apply by serverBase due to we have not split node api from server plugin system. So it will be done in future